projects/storefrontlib/src/lib/cms-lib/cms-lib.module.ts

```diff
@@ -13,6 +13,8 @@ import { SearchBoxModule } from './search-box/search-box.module';
 import { MiniCartModule } from './mini-cart/mini-cart.module';
 import { SiteContextSelectorModule } from './site-context-selector/site-context-selector.module';
 import { AddressBookModule } from './address-book/address-book.module';
+import { ProductListModule } from '../product';
```

Comment:
this import should be more specific, i.e `../product/foo/bar/product-list.module`